arm/Microsoft.Network/virtualNetworks/deploy.bicep

```diff
@@ -135,7 +152,7 @@ resource virtualNetwork_lock 'Microsoft.Authorization/locks@2016-09-01' = if (lo
 }
 
 resource appServiceEnvironment_diagnosticSettings 'Microsoft.Insights/diagnosticSettings@2017-05-01-preview' = if (!empty(diagnosticStorageAccountId) || !empty(workspaceId) || !empty(eventHubAuthorizationRuleId) || !empty(eventHubName)) {
-  name: '${vNetName}-diagnosticSettings'
+  name: '${name}-diagnosticSettings'
```

Comment:
update please to `name: '${virtualNetwork.name}-diagnosticSettings'`

Reply:
done